modules/javafx.controls/src/main/java/javafx/scene/control/ComboBox.java

```diff
@@ -123,10 +123,11 @@
  * a different type is specified and the ComboBox is to be editable, it is
  * necessary to specify a custom {@link StringConverter}.
  *
- * <h2>A warning about inserting Nodes into the ComboBox items list</h2>
+ * <h2>Warning: Nodes should not be inserted directly into the ComboBox items list</h2>
```

Comment:
Can you also add the bulleted list of "Important points to note" to `ComboBox`?

Reply:
Added the bulleted list.